src/Altinn.App.Core/Internal/Notifications/Email/IEmailNotificationClient.cs

```diff
@@ -0,0 +1,18 @@
+using Altinn.App.Core.Models.Notifications.Email;
+
+namespace Altinn.App.Core.Internal.Notifications.Email;
+
+/// <summary>
+/// Client for managing Altinn email notifications
+/// </summary>
+public interface IEmailNotificationClient
```

Comment:
Sorry I didn't see this before, but why is the public interface (that I assume we expect app developers to use) in the internal namespace?

Reply:
I've wondered that as well, there are tons of other public interfaces in the internal folder. I asked about this at some point but didn't really get an answer (i.e. seemed accidenal or atleast not intentional)

Reply:
I would hope those interfaces are not intended for direct usage in app code.

Reply:
Having "pubternal" stuff is not unprecedented, but there is a lot of stuff in there..
Notification interfaces are atleast intended to be public. Me making the implementation classes internal is also a deviation from the convention here..

Reply:
I'm not suggesting to make the interface internal, but to find a place for it where it isn't in a namespace named "Internal"